### PR TITLE
Generate a variable dictionary and values

### PR DESCRIPTION
**Pull request by lfc-256**
This dictionary include all parameters that we need to use in Matlab and Scilab.

Resolves-Issue #9

### CODE REVIEW

**Review by jcontrolresearch — CHANGES_REQUESTED** [review]
Please include a header to provide information about dictionary, for instance:

image

split parameters on control parameters/ vehicle parameters/ trailer parameters as comments and variables in each section

**Comment by lfc-256** [comment]
The changes to provide better organization of information and remove scripts for Scilab have been made.

**Review by raul-rd — APPROVED** [review]

**Review by SaraML — APPROVED** [review]

**Comment by lfc-256** [comment]
Hi Julio, I made the changes  and this generate conflicts with the merge pull request. What can I do? Thanks!!!

**Review by jcontrolresearch — CHANGES_REQUESTED** [review]
Use %% to header comment in order to define scrip information to future script external calls

**Comment by jcontrolresearch** [comment]
> Hi Julio, I made the changes and this generate conflicts with the merge pull request. What can I do? Thanks!!!

@lfc-256 please click on highlighted section to see solution

image

File: Scripts/variable_dictionary.m [hunk 1]
```diff
@@ -1,5 +1,20 @@
+%{
```
Comment (jcontrolresearch) [inline Scripts/variable_dictionary.m]: Here must be %%

File: Scripts/variable_dictionary.m [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+This scripts provides a variable dictionary with values of control
+parameters, vehicle parameters and trailer parameters that are used in the
+simulation .
+%}
```
Comment (jcontrolresearch) [inline Scripts/variable_dictionary.m]: Here to... %% please see [Code section spec Matlab](https://www.mathworks.com/help/matlab/matlab_prog/run-sections-of-programs.html)

**Review by jcontrolresearch — CHANGES_REQUESTED** [review]
Team @HugoLeyvaOax , @SaraML , @raul-rd  please take a look in [code section for matlab scritps](https://www.mathworks.com/help/matlab/matlab_prog/run-sections-of-programs.html) for further develop scripts

**Comment by HugoLeyvaOax** [comment]
Very Well

**Review by raul-rd — APPROVED** [review]

**Review by raul-rd — APPROVED** [review]

**Comment by lfc-256** [comment]
Thanks! I made the changes and follow the next steps [solution](https://github.com/githubteacher/github-for-developers-sept-2015/issues/648#issuecomment-144488223) and it worked.

**Review by raul-rd — APPROVED** [review]

**Review by SaraML — APPROVED** [review]

File: Scripts/variable_dictionary.m [hunk 3]
```diff
@@ -0,0 +1,16 @@
+%% This scripts provides a variable dictionary with values of control parameters, vehicle parameters and trailer parameters that are used in the simulation.
+
```
Comment (jcontrolresearch) [inline Scripts/variable_dictionary.m:7]: you forgot close the header with %% :)

**Review by jcontrolresearch — CHANGES_REQUESTED** [review]
Follow this structure :)

[here](https://github.com/lfc-256/ControlBDA/pull/18#pullrequestreview-417862759)

**Review by jcontrolresearch — APPROVED** [review]
Great !

**Review by SaraML — APPROVED** [review]

**Review by raul-rd — APPROVED** [review]

**Pull request merged**